lib/core/api.js

```diff
@@ -86,139 +88,135 @@ api.setHosts = function(hosts) {
+      rollbar.configure({
         payload: {
           person: {
             id: data.userid,
             email: user.username,
             username: user.username,
-          }
-        }
+          },
+        },
       });
     }
     return cb(null, data);
   });
 };
 
-api.user.loginExtended = function(user, options, cb){
+api.user.loginExtended = (user, options, cb) => {
   async.series([
     api.user.login.bind(null, user, options),
     api.user.loggedInProfile,
-    api.user.getUploadGroups
+    api.user.getUploadGroups,
   ], cb);
 };
 
-api.user.account = function(cb) {
+api.user.account = (cb) => {
   api.log('GET /auth/user');
-  tidepool.getCurrentUser(function(err, user) {
+  tidepool.getCurrentUser((err, user) => {
     // the rewire plugin messes with default export in tests
-    if (rollbar) {
-      rollbar.configure && rollbar.configure({
+    if (rollbar && rollbar.configure) {
+      rollbar.configure({
         payload: {
           person: {
             id: user.userid,
             email: user.username,
             username: user.username,
-          }
-        }
+          },
+        },
       });
     }
     cb(err, user);
   });
 };
 
-api.user.loggedInProfile = function(cb) {
-  api.log('GET /metadata/' + tidepool.getUserId() + '/profile');
-  tidepool.findProfile(tidepool.getUserId(), function(err, profile) {
-      if (err) {
-        return cb(err);
-      }
-      return cb(null, profile);
+api.user.loggedInProfile = (cb) => {
+  api.log(`GET /metadata/${tidepool.getUserId()}/profile`);
+  tidepool.findProfile(tidepool.getUserId(), (err, profile) => {
+    if (err) {
+      return cb(err);
+    }
+    return cb(null, profile);
   });
 };
 
-api.user.profile = function(userId, cb) {
-  api.log('GET /metadata/' + userId + '/profile');
-  tidepool.findProfile(userId, function(err, profile) {
+api.user.profile = (userId, cb) => {
+  api.log(`GET /metadata/${userId}/profile`);
+  tidepool.findProfile(userId, (err, profile) => {
     if (err) {
       return cb(err);
     }
     return cb(null, profile);
   });
 };
 
-api.user.addProfile = function(userId, profile, cb){
-  api.log('PUT /metadata/' + userId + '/profile');
-  tidepool.addOrUpdateProfile(userId, profile, function(err, response){
-    if(err){
+api.user.addProfile = (userId, profile, cb) => {
+  api.log(`PUT /metadata/${userId}/profile`);
+  tidepool.addOrUpdateProfile(userId, profile, (err, response) => {
+    if (err) {
       return cb(err);
     }
     return cb(null, response);
   });
 };
 
-api.user.updateProfile = function(userId, updates, cb){
-  api.user.profile(userId, function(err, profile){
-    if(err){
+api.user.updateProfile = (userId, updates, cb) => {
+  api.user.profile(userId, (err, profile) => {
+    if (err) {
       return cb(err);
     }
-    var currentEmail = _.get(profile, 'emails[0]');
+    const currentEmail = _.get(profile, 'emails[0]');
     // Note: we can't use var newProfile = actionUtils.mergeProfileUpdates here,
```

Comment:
Is this comment still relevant? Isn't this ES6 now?

Reply:
Ah, good call. will make the change.